### PR TITLE
Doc adjustments for v1

### PR DESCRIPTION
**Pull request by jwpereira**
With us heading into the marketplace real soon, I want to make sure the README will reflect the v1 status and latest discussions on naming 🤔

### CODE REVIEW

**Review by acampbel — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Setup MATLAB](#set-up-matlab) action. The action sets up the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
 
-The **Set Up MATLAB** action is not supported on [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runners. Currently, it is available only for public projects. It does not install transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+The **Setup MATLAB** action is not supported on [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runners. Currently, it is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
```
Comment (mcafaro) [inline README.md:5]: Not necessarily needed for this release, but I wonder if we should include a sentence after "The Setup MATLAB action is not supported..." sentence that says something like "To use MATLAB with a self-hosted agent, you can preintall MATLAB using the MATLAB installer."

File: README.md [hunk 2]
```diff
@@ -41,24 +41,24 @@ jobs:
...
 ```
 
 ## Set Up MATLAB
-When you define your workflow in the `.github/workflows` directory of your repository, specify the **Set Up MATLAB** action as `matlab-actions/setup-matlab@v0`. The action accepts an optional input.
+When you define your workflow in the `.github/workflows` directory of your repository, specify the **Setup MATLAB** action as `matlab-actions/setup-matlab@v1`. The action accepts an optional input.
 
 | Input     | Description |
 |-----------|-------------|
-| `release` | (Optional) MATLAB release to install. You can specify R2020a or a later release. If you do not specify `release`, the action installs the latest release of MATLAB.<br/>**Example**: `R2020a`
+| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2020a`
```
Comment (mcafaro) [inline README.md:58]:
```suggestion
| `release` | (Optional) MATLAB release to set up. You can specify R2020a or a later release. If you do not specify `release`, the action sets up the latest release of MATLAB.<br/>**Example**: `R2021a`
```

I like always like promoting our latest release when we can :)

**Review by mcafaro — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md:58]: +1

**Review by mw-hrastega — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jwpereira) [reply README.md:5]: Interesting and I like that 🤔 😄 
@mw-hrastega would you happen to have a phrase in mind for something like this 🤔 
Else, we can defer it to a future release 😄

File: README.md (hunk 2 above)
Comment (jwpereira) [reply README.md]: Me too; I directly committed your suggestion (neato feature in GitHub I must say 😄)

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md:5]: I am not sure if we are on the same page. Why "can"? What is the other options? 

To use MATLAB with a self-hosted agent, you can preinstall MATLAB using the MATLAB installer.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
-# Action for Installing MATLAB on GitHub-Hosted Runner
+# Action for Setting Up MATLAB on GitHub-Hosted Runner
 
-Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Set Up MATLAB](#set-up-matlab) action. The action installs the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB. 
+Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Setup MATLAB](#set-up-matlab) action. The action sets up the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
```
Comment (mw-hrastega) [inline README.md:3]: Won't the link break here?

[Setup MATLAB](#set-up-matlab)

Comment (jwpereira) [reply README.md:3]: RIP I merged right before reading this 😂 
I'm going to follow this PR up with another one 😂

Comment (jwpereira) [reply README.md:3]: Okay I just checked, I think the link still works 🤔 
Just checked in main 🤔

File: README.md (hunk 1 above)
Comment (jwpereira) [reply README.md:5]: Okay we can PR this separately in time 😂

**Pull request merged**